controllers/certificatesigningrequests_controller.go

```diff
@@ -75,15 +75,16 @@ func NewCertificateSigningRequestsReconciler(mgr manager.Manager, clusterConfig
 // Reconcile is part of the main kubernetes reconciliation loop which reads that state of the cluster for a
 // CertificateSigningRequests object and aims to move the current state of the cluster closer to the desired state.
 func (r *certificateSigningRequestsReconciler) Reconcile(ctx context.Context,
-	req ctrl.Request) (res ctrl.Result, reconcileErr error) {
+	req ctrl.Request) (result ctrl.Result, reconcileErr error) {
```

Comment:
Why are you making this changes?

In general `[condition] Send an event on OperatorCond failure ` belongs to the commit where you are introducing the `[wmco] Opt in to Upgradeable operator condition. Look at this you are logging errors, so it should be done in the commit where you are introducing the feature  and it will also reduce code churn.